canvas_cli/utils/validators/manifest_schema.py

```diff
@@ -52,7 +52,6 @@
         "name",
         "description",
         "components",
-        "tags",
```

Comment:
do we need to require "tags"? I do see a lot of validation around values that are included here, but I have never seen tags used and I don't know that we have immediate plans to start recommending their use.

Reply:
i think the plan is to use them for searchability/findability on the marketing website. might want to check with @andhines on that one to see if they are actually being used that way

Reply:
ok. I will leave it as required.